programs/eosio-launcher/main.cpp

```diff
@@ -1157,27 +1164,18 @@ launcher_def::write_logging_config_file(tn_node_def &node) {
 
 void
 launcher_def::init_genesis () {
-  bfs::path genesis_path = bfs::current_path() / "genesis.json";
-   bfs::ifstream src(genesis_path);
-   if (!src.good()) {
+   const bfs::path genesis_path = genesis.is_complete() ? genesis : bfs::current_path() / genesis;
```

Comment:
While we're at it, let's work around the boost defect that prevents `("genesis,g",bpo::value<bfs::path>(&genesis)` from working if the argument has spaces in the name.